tests/scorers/test_coherence_scorer.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from tests.scorers.test_utils import TINY_MODEL_PATHS
+from weave.scorers.coherence_scorer import WeaveCoherenceScorerV1
+
+
+@pytest.fixture
+def weave_coherence_scorer():
+    """Fixture to return a WeaveCoherenceScorer instance."""
+    scorer = WeaveCoherenceScorerV1(
+        model_name_or_path=TINY_MODEL_PATHS["coherence_scorer"],
+        device="cpu",
+    )
+    return scorer
+
+
+def test_score_messages(weave_coherence_scorer):
+    """Test score_messages with a coherent response."""
+    query = "This is a test prompt."
+    output = "This is a coherent response."
+    result = weave_coherence_scorer._score_messages(query, output)
+    # Check that the pydantic model has the expected attributes.
+    assert hasattr(result, "passed")
+    assert hasattr(result, "extras")
```

Comment:
Instead of checking for existence, I would just check the values directly.  In this case, I think you check `result.passed is True`